src/dbnode/generated/proto/annotation/annotation.proto

```diff
@@ -3,11 +3,26 @@ syntax = "proto3";
 package annotation;
 
 message Payload {
-    MetricType metric_type   = 1;
-    bool handle_value_resets = 2;
+    SourceFormat source_format = 3;
+
+    // Used when source_format == OPEN_METRICS_FORMAT
+    OpenMetricsFamilyType open_metrics_family_type = 1;
+    bool open_metrics_handle_value_resets          = 2;
+
+    // Used when source_format == GRAPHITE_FORMAT
+    GraphiteType graphite_type = 4;
 }
 
-enum MetricType {
+enum SourceFormat {
+    // OPEN_METRICS_FORMAT also includes Prometheus data.
+    // Using 0 for OPEN_METRICS_FORMAT because only Prometheus types were being stored before.
+    OPEN_METRICS_FORMAT = 0;
+
+    // Graphite/statsd.
+    GRAPHITE_FORMAT = 1;
```

Comment:
nit: maybe these could be without `_FORMAT` suffix?

Reply:
👍